spec/domain/monitor/dimensions_spec.rb

```diff
@@ -9,15 +9,15 @@
 
   before { allow(GovukStatsd).to receive(:count) }
 
-  it 'sends the total number of base_paths' do
+  it 'sends StatsD counter of base_paths' do
```

Comment:
Sorry, I'm not clear how the commit message applies to the changes to the spec.

Reply:
As per our conversation I have amended the commit notes explaining that
this names are consistent with other test names for the ETL monitoring.